### DISCUSSION TITLE
首页番剧列表混入国创内容

### DISCUSSION THREAD

**Discussion by sakowei** [the1812/Bilibili-Evolved#3535] [提问 / Q&A] [answered]
在开启“”清爽首页”组件后的番剧列表怎么混入了一些国创的内容？关掉组件和脚本就不会出现问题
![image](https://user-images.githubusercontent.com/41941069/181405864-84b01041-ab7d-4db1-812a-b53405b5635d.png)

**Comment by sakowei** [comment]
国创内容已画圈表示
![QwhUiHouJlvIL52RXYFJyGuMVbI1rW7R](https://user-images.githubusercontent.com/41941069/181406464-47e94e26-adc4-404e-a128-ee6fbf15ac4f.jpg)

**Comment by the1812** [comment, marked as answer]
应该是个 bug

**Reply by the1812** [reply]
转到 #3552 了
